training_config/transformer_squad.bert-base.jsonnet

```diff
@@ -0,0 +1,60 @@
+local transformer_model = "bert-base-cased";
+local epochs = 3;
+local batch_size = 8;
+
+// This defines the number of instances, not the number of questions. One question can end up as
+// multiple instances.
```

Comment:
You might note that we only need this for the learning rate scheduler.  It's pretty surprising that you would have to hard-code this.

Reply:
We should make a change where the learning rate scheduler can find out about `num_steps_per_epoch` automatically, at least when you have a non-lazy reader. This thing with the four different files is dumb. I'm not sure I want to do it in this PR though, since it'll go across repos, and this one is already pretty long.

Reply:
I'm happy to take an issue based on this.

Reply:
https://github.com/allenai/allennlp/issues/3707